content/en/blog/_posts/2023-10-10-read-write-once-pod-access-mode-ga.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: blog
+title: "Kubernetes 1.29: Single Pod Access Mode for PersistentVolumes Graduates to GA/Stable"
+date: 2023-10-10
+slug: read-write-once-pod-access-mode-ga
+---
+
+**Author:** Chris Henzie (Google)
+
+With the release of Kubernetes 1.29 the ReadWriteOncePod feature has graduated
+to GA. In this blog post, we'll take a closer look at this feature and what it
+does.
+
+## What is ReadWriteOncePod?
+
+ReadWriteOncePod is a new access mode for
```

Comment:
```suggestion
ReadWriteOncePod is an access mode for
```
It can be new, or it can have been beta since v1.27; I don't think it can be both.